stored_procedures/create_run_record.sql

```diff
@@ -53,7 +51,6 @@ BEGIN
       SELECT id FROM `automation_systems`
       WHERE automation_system_name = input_automation_system_name
     ),
-    input_system_run_id,
```

Comment:
Is both `start_time` and `created_at` needed? Does `start_time` get updated later? And should `updated_at` be set to now() on creation, or should it be left empty?

Reply:
Not sure. One is the PC time the run started and the other database insert time. If the first thing they do on starting the run is insert the row maybe it's fine without.
What's the standard with updated_at? does it get set same as created_at on first insert normally?